Tutorial.md

```diff
@@ -1,7 +1,8 @@
-# API Blueprint Tutorial
-Welcome to the API Blueprint Tutorial! In this tutorial you will learn all about all the key features of the API Blueprint Language on the example of an API for our imaginary industry-changing Gist service.
+# API Blueprint Basic Tutorial
 
-Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set some expectations let's provide following functionality in our API: *List Gists, Retrieve a Single Gist, Create a Gist, Edit a Gist, Star a Gist, Unstar a Gist, Check if a Gist is starred and Delete a Gist.*
+Welcome to the API Blueprint Tutorial! In this tutorial you will learn about the basics to getting started with the API Blueprint language.
```

Comment:
Unity: I feel you are switching pronouns quite often is it "you learning" or "us learning" the API Blueprint? There is 18 occurrences of "our", 25 occurrences of "you", and one occurrence of "yours". 

I would say the tutorial should decide at what person it is "talking to" and stay with it. By that I mean it is either 

"In this tutorial you will learn" & "your blueprint"

Or

"In this tutorial we will learn" & "our blueprint"

Not the mix of the two. Personally I vote for the former as it is a little less formal